test/integration/generated/additionalProperties/package.json

```diff
@@ -84,5 +84,11 @@
     "docs": "echo skipped"
   },
   "sideEffects": false,
-  "autoPublish": true
+  "autoPublish": true,
+  "//sampleConfiguration": {
```

Comment:
I don't see any samples generated for this integration test. Why are we adding this sample metadata config?

Reply:
changed it